Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/cartons.yml

```diff
@@ -73,12 +79,15 @@
     sprite: Objects/Consumable/Smokeables/Cigarettes/Cartons/black.rsi
     layers:
       - state: closed
+        map: ["closeLayer"] #DeltaV change, fixed broken "open" layer
+      - state: open
+        map: ["openLayer"] #End DeltaV Change
```

Comment:
these layers get inherited so comment out the `layers: - state: closed` lines for the child prototypes instead